core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/store/dot-experiments-list-store.ts

```diff
@@ -81,17 +81,33 @@ export class DotExperimentsListStore
     implements OnStoreInit
 {
     // Selectors
-    readonly getExperimentsFilteredAndGroupedByStatus$ = this.select(
-        ({ experiments, filterStatus }) =>
-            experiments
-                .filter((experiment) => filterStatus.includes(experiment.status))
-                .reduce<GroupedExperimentByStatus>((group, experiment) => {
-                    group[experiment.status] = group[experiment.status] ?? [];
-                    group[experiment.status].push(experiment);
+    readonly getExperimentsFilteredAndGroupedByStatus$: Observable<GroupedExperimentByStatus[]> =
+        this.select(({ experiments, filterStatus }) => {
+            const grouped: GroupedExperimentByStatus[] = [];
 
-                    return group;
-                }, <GroupedExperimentByStatus>{})
-    );
+            if (experiments.length) {
+                Object.keys(DotExperimentStatusList).forEach((key) =>
+                    grouped.push({ status: DotExperimentStatusList[key], experiments: [] })
+                );
+
+                experiments
+                    .filter((experiment) => filterStatus.includes(experiment.status))
+                    .forEach((experiment) => {
+                        const status = grouped.find((item) => item.status === experiment.status);
+
+                        if (status) {
+                            status.experiments.push(experiment);
```

Comment:
What do we use this `status` constant for? I think we are not using or returning it.

Reply:
I will change the name to `statusGroup` basically if is present we just need to push the experiment.